public/js/sequence_modal.js

```diff
@@ -160,6 +150,11 @@ class SequenceViewer extends React.Component {
         footer: false,
       },
     });
-    widget.hideFormatSelector();
+    setTimeout(function() {
+      if (widget._headerDiv != null) {
+        widget.hideFormatSelector();
+        return;
+      }
+    }, 200);
```

Comment:
What's happening here?

Reply:
There is a race condition occurring when we invoke hideFormatSelector before _headerDiv is set.